maap/utils/requests_utils.py

```diff
@@ -1,12 +1,12 @@
 import os
 
-from maap.config_reader import ConfigReader
+from maap.config_reader import MaapConfig
 
 
 class RequestsUtils:
     @staticmethod
     def generate_dps_headers(content_type=None):
-        config = ConfigReader()
+        config = MaapConfig()
```

Comment:
Pass a `MAAP` instance into this method, then do this (or pass a `MaapConfig` instance in, rather than a `MAAP` instance):

```suggestion
        config = maap.config
```

Also, this is Python, not Java, so I'd suggest eliminating the `RequestUtils` class since it contains only static methods. Make the methods functions.